.github/workflows/e2e.yml

```diff
@@ -74,8 +77,6 @@ jobs:
 
       - name: Test
         uses: ./.github/actions/test-e2e
-        env:
-          NEPTUNE_API_TOKEN: ${{secrets.E2E_NEPTUNE_API_TOKEN}}
```

Comment:
why `e2e_management` doesn't need `NEPTUNE_API_TOKEN` all of the sudden?

Reply:
It was moved to a common `env` section. And I wanted to made it a way cleaner which API key are we going to use.